packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts

```diff
@@ -182,11 +180,11 @@ $btn-drag-image: '../img/grab_light.svg';
 $input-bg: $white;
 $input-bg-disabled: $gray-5;
 
-$input-color: $dark-5;
+$input-color: $dark-2;
```

Comment:
seems like big change?

Reply:
Actually nothing has changed from master. Before I removed some dark variables. So i renamed dark-5 to dark-2.